lib/circuitbox/circuit_breaker.rb

```diff
@@ -17,8 +17,7 @@ class CircuitBreaker
     # `sleep_window`      - seconds to sleep the circuit
     # `volume_threshold`  - number of requests before error rate calculation occurs
     # `error_threshold`   - percentage of failed requests needed to trip circuit
-    # `timeout_seconds`   - seconds until it will timeout the request
```

Comment:
Since this is a breaking change, we should add a warning here if someone is still passing it.  We can probably point them at the changelog or some doc on GitHub to inform them.